core/src/default_parameters.rs

```diff
@@ -158,3 +158,11 @@ pub fn print_contract_hashes() {
         encoded_hash(update_ergo_tree_bytes)
     );
 }
+
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn print_contract_hashes() {
+        super::print_contract_hashes();
```

Comment:
I was thinking of hard-coding the hashes in the unit tests.
The workflow I have in mind for checking a contract's hash in the unit test is the following:
- Let's say we have a PR that changes a contract (P2S string in defaults);
- On PR review, I want to make sure that the updated contract is the same contract that is defined in EIP (or somewhere else); 
- I check that the updated hash in the unit test is the same hash as in EIP(or somewhere else);

Reply:
Sounds good, I've added the hashes in.